### PR TITLE
update soft asserts

### PR DESCRIPTION
**Pull request by czue**
figured it doesn't make sense for me to own these anymore.

killed a few that haven't ever triggered.

also a few other things uncovered by grep.

@calellowitz - easiest by commit

### CODE REVIEW

File: deployment/apache/local_apache.conf [hunk 1]
```diff
@@ -7,9 +7,9 @@
     SetEnv proxy-nokeepalive 1
 
     # uncomment these lines to enable apache to serve static files
-    # (replace /home/czue/src/commcare-hq with the path to your local HQ install
-    # Alias /static /home/czue/src/commcare-hq/staticfiles/
-    # <Directory /home/czue/src/commcare-hq/staticfiles>
+    # (replace /home/your_username/src/commcare-hq with the path to your local HQ install
+    # Alias /static /home/your_username/src/commcare-hq/staticfiles/
```
Comment (benrudolph) [inline deployment/apache/local_apache.conf]: haha

Comment (benrudolph) [inline deployment/apache/local_apache.conf]: could probably kill this whole thing

Comment (czue) [inline deployment/apache/local_apache.conf]: why?

File: corehq/ex-submodules/pillow_retry/tests.py [hunk 2]
```diff
@@ -116,7 +115,7 @@ def test_get_errors_to_process(self):
         # current_attempt < setting.PILLOW_RETRY_QUEUE_MAX_PROCESSING_ATTEMPTS
         date = datetime.utcnow()
         for i in range(0, 5):
-            error = create_error({'id': i}, attempts=i+1)
+            error = create_error(_change(id=i), attempts=i+1)
```
Comment (dimagimon) [inline corehq/ex-submodules/pillow_retry/tests.py]: E226 missing whitespace around arithmetic operator

File: deployment/apache/local_apache.conf (hunk 1 above)
Comment (benrudolph) [inline deployment/apache/local_apache.conf]: we don't use apache anymore and doubt anyone is trying to keep this up to date. but suppose could be useful for someone who really wants to deploy commcare on apache

Comment (czue) [inline deployment/apache/local_apache.conf]: oh right, sorry that makes sense. will remove.

Comment (czue) [inline deployment/apache/local_apache.conf]: https://github.com/dimagi/commcare-hq/pull/15175/commits/8a1253c2b33e4829b421e5335af0dd0a5033d69e

**Comment by czue** [comment]
:white_check_mark:

**Pull request merged**